people/birkenfeld.toml

```diff
@@ -2,5 +2,3 @@ name = "Georg Brandl"
 github = "birkenfeld"
 github-id = 144359
 
-[permissions]
-bors.clippy.review = true
```

Comment:
needs a mail address, too (according to CI)

Reply:
oh oops

Reply:
Done